apps/s_client.c

```diff
@@ -2172,6 +2172,15 @@ int s_client_main(int argc, char **argv)
              * HTTP/d.d ddd Reason text\r\n
              */
             mbuf_len = BIO_gets(fbio, mbuf, BUFSIZZ);
+            if (mbuf_len < sizeof("HTTP/1.0 200") - 1) {
```

Comment:
Yes, I understand what you expressed. Further more, I found a problem with using sizeof that reported by Travis-CI, the 'signed/unsigned' stuff (need explicit cast for some compilers)

For simplicity, let's use strlen instead...